readme.md

```diff
@@ -422,7 +422,16 @@ This repository is licensed under the Apache 2.0 License, except for the [server
 
 ## Development
 
-Read the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) documentation for more information about developing and extending plugins.
+This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/integrate/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/integrate/plugins/developer-setup/) for more information about developing and extending plugins.
+
+### Releasing new versions
+
+The version of a plugin is determined at compile time, automatically populating a `version` field in the [plugin manifest](plugin.json):
+* If the current commit matches a tag, the version will match after stripping any leading `v`, e.g. `1.3.1`.
+* Otherwise, the version will combine the nearest tag with `git rev-parse --short HEAD`, e.g. `1.3.1+d06e53e1`.
+* If there is no version tag, an empty version will be combined with the short hash, e.g. `0.0.0+76081421`.
+
+To disable this behaviour, manually populate and maintain the `version` field.
```

Comment:
Wondering how version bumps work in this case. Would we just not have version bump PRs anymore? The tag is what decides the version now IIUC. @hanzei Thoughts on this? We typically use version bump PRs as a means to publicly discuss things about the releases

Reply:
@mickmister I like version bump PRs to as a sync point for all parties involved. Hence, I'm leaning towards keeping the process as it is for major plugins like JIRA.